[2025-02-27T19:21:45.549+0000] {processor.py:186} INFO - Started process (PID=2642) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:21:45.551+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:21:45.555+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:21:45.554+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:21:45.561+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:21:45.560+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:21:45.561+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:21:45.561+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:21:45.561+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:21:45.580+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2025-02-27T19:23:27.318+0000] {processor.py:186} INFO - Started process (PID=2736) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:23:27.321+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:23:27.325+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:23:27.324+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:23:27.331+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:23:27.329+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:23:27.331+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:23:27.331+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:23:27.331+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:23:27.350+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2025-02-27T19:24:27.990+0000] {processor.py:186} INFO - Started process (PID=2806) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:24:27.992+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:24:27.998+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:24:27.997+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:24:28.005+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:24:28.004+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:24:28.006+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:24:28.005+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:24:28.006+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:24:28.026+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2025-02-27T19:25:29.186+0000] {processor.py:186} INFO - Started process (PID=2866) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:25:29.188+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:25:29.193+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:25:29.192+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:25:29.199+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:25:29.198+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:25:29.199+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:25:29.199+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:25:29.200+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:25:29.218+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2025-02-27T19:26:34.675+0000] {processor.py:186} INFO - Started process (PID=2932) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:26:34.677+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:26:34.681+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:26:34.680+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:26:34.686+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:26:34.685+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:26:34.686+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:26:34.686+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:26:34.686+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:26:34.705+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2025-02-27T19:27:38.990+0000] {processor.py:186} INFO - Started process (PID=2996) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:27:38.992+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:27:38.995+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:27:38.995+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:27:39.000+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:27:39.000+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:27:39.001+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:27:39.001+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:27:39.001+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:27:39.017+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2025-02-27T19:28:38.088+0000] {processor.py:186} INFO - Started process (PID=3074) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:28:38.090+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:28:38.093+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:28:38.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:28:38.098+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:28:38.098+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:28:38.099+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:28:38.099+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:28:38.099+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:28:38.119+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2025-02-27T19:29:37.055+0000] {processor.py:186} INFO - Started process (PID=3141) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:29:37.058+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:29:37.063+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:29:37.062+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:29:37.069+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:29:37.068+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:29:37.071+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:29:37.070+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:29:37.071+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:29:37.094+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2025-02-27T19:30:39.473+0000] {processor.py:186} INFO - Started process (PID=3206) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:30:39.476+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:30:39.480+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:30:39.479+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:30:39.485+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:30:39.484+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:30:39.485+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:30:39.485+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:30:39.485+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:30:39.501+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2025-02-27T19:31:44.277+0000] {processor.py:186} INFO - Started process (PID=3268) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:31:44.280+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:31:44.284+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:31:44.283+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:31:44.290+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:31:44.289+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:31:44.290+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:31:44.290+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:31:44.291+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:31:44.310+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2025-02-27T19:32:58.596+0000] {processor.py:186} INFO - Started process (PID=3343) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:32:58.599+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:32:58.603+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:32:58.602+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:32:58.612+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:32:58.610+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:32:58.613+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:32:58.613+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:32:58.613+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:32:58.634+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2025-02-27T19:34:02.403+0000] {processor.py:186} INFO - Started process (PID=3406) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:34:02.405+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:34:02.409+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:34:02.408+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:34:02.414+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:34:02.413+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:34:02.414+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:34:02.414+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:34:02.415+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:34:02.431+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2025-02-27T19:35:05.830+0000] {processor.py:186} INFO - Started process (PID=3471) to work on /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:35:05.833+0000] {processor.py:914} INFO - Processing file /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-27T19:35:05.837+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:35:05.836+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:35:05.842+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:35:05.841+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-27T19:35:05.842+0000] {logging_mixin.py:190} INFO - [2025-02-27T19:35:05.842+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-27T19:35:05.843+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-27T19:35:05.859+0000] {processor.py:208} INFO - Processing /home/sravan2418/airflow-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
